cas-server-webapp-support/src/main/java/org/jasig/cas/web/support/CookieRetrievingCookieGenerator.java

```diff
@@ -103,9 +103,14 @@ public void addCookie(final HttpServletRequest request, final HttpServletRespons
      * @return the cookie value
      */
     public String retrieveCookieValue(final HttpServletRequest request) {
-        final Cookie cookie = org.springframework.web.util.WebUtils.getCookie(
-                request, getCookieName());
-        return cookie == null ? null : this.casCookieValueManager.obtainCookieValue(cookie, request);
+        try {
+            final Cookie cookie = org.springframework.web.util.WebUtils.getCookie(
+                    request, getCookieName());
+            return cookie == null ? null : this.casCookieValueManager.obtainCookieValue(cookie, request);
+        } catch (final Exception e) {
+            logger.debug(e.getMessage(), e);

```

Comment:
I found out that there are many other keys out there that we dont necessarily have anything for them in our bundles, and dont want to anyway. Spring and Spring Security for instance try to load off the description of their error messages from the app bundle, and to generate a warn for very single statement is just noise. If the key is not found, the key itself is used verbatim and to investigate, an accepted approach is to always turn on DEBUG.